### DISCUSSION TITLE
Start full screen on Linux

### DISCUSSION THREAD

**Discussion by ThePsychoDave** [KlipperScreen/KlipperScreen#773] [General]
Hi.

I have installed Klipper on a laptop running Mint linux. Since the laptop has a touch screen I installed Klipper Screen on it and set it to auto start. It auto starts and works well but it does not start in full screen and I have to manually select it at every boot. Linux is not my strongest side, but is there a way to start Klipper Screen in full screen, maybe by editing screen.py?

**Comment by alfrix** [comment]
if you want to run only KlipperScreen, then you may want to do `sudo systemctl set-default multi-user.target` as explained in the [docs](https://klipperscreen.readthedocs.io/en/latest/Installation/#klipperscreen-installation)
that being said, i will make KlipperScreen set the fullscreen flag by default

**Reply by ThePsychoDave** [reply]
I tried it and then I only ends up at an command prompt where I have to log in, when I have logged in nothing else happens.

**Comment by alfrix** [comment]
https://klipperscreen.readthedocs.io/en/latest/Troubleshooting/
